x-pack/legacy/plugins/ml/public/jobs/new_job_new/recognize/page.tsx

```diff
@@ -57,6 +59,8 @@ interface PageProps {
   existingGroupIds: string[];
 }
 
+export type JobOverrides = Record<JobId, Partial<JobOverride>>;
```

Comment:
is `Partial` needed here, `JobOverride` is already a partial `Job`

Reply:
Fixed in https://github.com/elastic/kibana/pull/49310/commits/d7ea3c628d2d4059974f61ef943aa256b492cafd